application/CMakeLists.txt

```diff
@@ -106,9 +106,11 @@ if (F3D_MODULE_EXR)
   target_compile_definitions(f3d PRIVATE F3D_MODULE_EXR)
 endif ()
 
+if(VTK_VERSION VERSION_LESS 9.3.20240914)
 # Headless EGL build
-if (VTK_OPENGL_HAS_EGL)
-  target_compile_definitions(f3d PRIVATE F3D_HEADLESS_BUILD)
+  if (VTK_OPENGL_HAS_EGL)
+    target_compile_definitions(f3d PRIVATE F3D_HEADLESS_BUILD)
```

Comment:
I think this variable was used to determine the render window to instantiate?
How is it working now? EGL is always available?

Reply:
see discord